src/IssueViz.Security.UnitTests/OpenInIdeHotspots/OpenHotstpotInIdeHandlerTests.cs

```diff
@@ -1,4 +1,5 @@
-﻿using Microsoft.VisualStudio.TestTools.UnitTesting;
+﻿using FluentAssertions;
```

Comment:
This is quite interesting. I wonder how it was missed

Reply:
The rule as a scope All, and that causes that it is not executed on test code, because SC/SQ is not ready for properly going on full analysis with test code.

You can do this, please create a new issue:
https://github.com/SonarSource/sonar-dotnet/blob/62d473bf63361ef455375e2e82147698e3c80589/.editorconfig#L230

Reply:
https://github.com/SonarSource/sonarlint-visualstudio/issues/5397